src/repositories/donationRepository.js

```diff
@@ -31,4 +31,22 @@ const updateBridgePaymentAuthorizedTxHash = async (
   );
 };
 
-module.exports = { updateBridgePaymentExecutedTxHash, updateBridgePaymentAuthorizedTxHash };
+const isAllDonationsPaidOutForMilestoneAndTxHash = async (app, { txHash, milestoneId }) => {
+  const donationModel = app.service('donations').Model;
+  const notPaidOutDonationsCount = await donationModel.count({
+    txHash,
+    status: DonationStatus.PAID,
+    ownerTypeId: milestoneId,
+    // after payout  bridgePaymentExecutedTxHash field should be filled
+    bridgePaymentExecutedTxHash: {
+      $exists: false,
+    },
+  });
+  return notPaidOutDonationsCount === 0;
+};
+
+module.exports = {
+  updateBridgePaymentExecutedTxHash,
+  updateBridgePaymentAuthorizedTxHash,
+  isAllDonationsPaidOutForMilestoneAndTxHash,
```

Comment:
Please choose simpler name, it's not need to have arguments' name in function name too

Reply:
@aminlatifi  done